### PR TITLE
update api version string

### PR DESCRIPTION
**Pull request by jasmineymlo**
update api version string to "2017-06-30"

### CODE REVIEW

**Comment by tameraw** [comment]
:shipit:

File: device/Microsoft.Azure.Devices.Client/Common/Api/ClientApiVersionHelper.cs [hunk 1]
```diff
@@ -5,7 +5,11 @@ namespace Microsoft.Azure.Devices.Client
 {
     class ClientApiVersionHelper
     {
-        public const string ApiVersionQueryString = "api-version=2016-11-14";
-        public const string ApiVersionString = "2016-11-14";
+        const string ApiVersionQueryPrefix = "api-version=";
+        const string ApiVersionNov2016 = "2016-11-14";
```
Comment (tameraw) [inline device/Microsoft.Azure.Devices.Client/Common/Api/ClientApiVersionHelper.cs]: >2016-11-14" [](start = 42, length = 11)

Will need to do the same for Service SDK as well.

**Comment by tameraw** [comment]
:shipit:

**Comment by anthonyvercolano** [comment]
:shipit:

**Comment by jasmineymlo** [comment]
did the same for Service SDK

**Pull request merged**